packages/dds/tree/src/simple-tree/core/allowedTypes.ts

```diff
@@ -637,8 +634,8 @@ export type TreeNodeFromImplicitAllowedTypes<
  * This type exists only to be linked from documentation to provide a single linkable place to document some details of
  * "Input" types and how they handle schema.
  *
- * When a schema is used to describe data which is an input into an API, the API is {@link https://en.wikipedia.org/wiki/Covariance_and_contravariance_(computer_science | contravariant}) over the schema.
```

Comment:
this link was broken (missing the ")") but also points to a redirect page. Updated.